### PR TITLE
Release 1.11.0 beta.8

### PR DESCRIPTION
**Pull request by yumikohey**

### CODE REVIEW

File: src/private/teams.ts [hunk 1]
```diff
@@ -51,13 +51,17 @@ export namespace teams {
...
+    if (!threadId) {
+      throw new Error('[teams.refreshSiteUrl] threadId cannot be null or empty');
+    }
+
     if (!callback) {
       throw new Error('[teams.refreshSiteUrl] Callback cannot be null');
     }
 
-    sendMessageToParent(threadId);
+    sendMessageToParent('teams.refreshSiteUrl', [threadId], callback);
```
Comment (ydogandjiev) [inline src/private/teams.ts:65]: There shouldn't be any actual code changes in the release branch. Please check those into master separately.

**Review by ydogandjiev — CHANGES_REQUESTED** [review]

File: src/private/teams.ts (hunk 1 above)
Comment (yumikohey) [reply src/private/teams.ts:65]: This line of code is in master code base :)

**Comment by sachingupta** [comment]
why are we update beta7 from beta 8?

**Comment by yumikohey** [comment]
> why are we update beta7 from beta 8?

As 1.11.0-beta.7 branch was already created

**Pull request closed without merging**